pkg/constants/env_variables.go

```diff
@@ -49,5 +49,6 @@ const (
 	// CSPLoginOrgID overrides the CSP default OrgID to which the user logs into, using CLI interactive login flow
 	// Note: More information regarding the CSP organizations can be found at
 	// https://docs.vmware.com/en/VMware-Cloud-services/services/Using-VMware-Cloud-Services/GUID-CF9E9318-B811-48CF-8499-9419997DC1F8.html
-	CSPLoginOrgID = "TANZU_CLI_CLOUD_SERVICES_ORGANIZATION_ID"
+	CSPLoginOrgID              = "TANZU_CLI_CLOUD_SERVICES_ORGANIZATION_ID"
+	TanzuCLIDBRefreshThreshold = "TANZU_CLI_DATABASE_REFRESH_THRESHOLD"
```

Comment:
Can you move this right after `ConfigVariablePluginDBCacheTTL` and use a similar naming such as `ConfigVariablePluginDBCacheRefreshThreshold = "TANZU_CLI_PLUGIN_DB_CACHE_REFRESH_THRESHOLD_HOURS"`

Reply:
Renamed and moved as suggested